docs/design/ovs-pipeline.md

```diff
@@ -1,1197 +1,1909 @@
+3. table=AntreaPolicyEgressRule, priority=44900,ip,nw_src=10.10.0.24 actions=conjunction(7,1/3)
+4. table=AntreaPolicyEgressRule, priority=44900,ip,nw_dst=10.10.0.25 actions=conjunction(7,2/3)
+5. table=AntreaPolicyEgressRule, priority=44900,tcp,tp_dst=3306 actions=conjunction(7,3/3)
+6. table=AntreaPolicyEgressRule, priority=44900,conj_id=7,ip actions=set_field:0x7->reg5,ct(commit,table=EgressMetric,zone=65520,exec(set_field:0x700000000/0xffffffff00000000->ct_label))
+7. table=AntreaPolicyEgressRule, priority=44899,ip,nw_src=10.10.0.24 actions=conjunction(5,1/2)
+8. table=AntreaPolicyEgressRule, priority=44899,ip actions=conjunction(5,2/2)
+9. table=AntreaPolicyEgressRule, priority=44899,conj_id=5 actions=set_field:0x5->reg3,set_field:0x400/0x400->reg0,goto_table:EgressMetric
+10. table=AntreaPolicyEgressRule, priority=44880,ip,nw_src=10.10.0.26 actions=conjunction(11,1/3)
+11. table=AntreaPolicyEgressRule, priority=44880,ip actions=conjunction(11,2/3)
+12. table=AntreaPolicyEgressRule, priority=44880,tcp,tp_dst=53 actions=conjunction(11,3/3)
+13. table=AntreaPolicyEgressRule, priority=44880,udp,tp_dst=53 actions=conjunction(11,3/3)
+14. table=AntreaPolicyEgressRule, priority=44880,conj_id=11,ip actions=set_field:0xb->reg5,ct(commit,table=EgressMetric,zone=65520,exec(set_field:0xb00000000/0xffffffff00000000->ct_label))
+15. table=AntreaPolicyEgressRule, priority=44879,ip actions=conjunction(13,2/2)
+16. table=AntreaPolicyEgressRule, priority=44879,ip,nw_src=10.10.0.26 actions=conjunction(13,1/2)
+17. table=AntreaPolicyEgressRule, priority=44879,conj_id=13,ip actions=set_field:0xd->reg5,ct(commit,table=EgressMetric,zone=65520,exec(set_field:0xd00000000/0xffffffff00000000->ct_label,set_field:0x80/0x80->ct_mark,set_field:0x40000000000000000/0xfff0000000000000000->ct_label))
+18. table=AntreaPolicyEgressRule, priority=0 actions=goto_table:EgressRule
 ```
 
-Similar to [K8s NetworkPolicy implementation](#egressruletable-50),
-AntreaPolicyEgressRuleTable also relies on the OVS built-in `conjunction` action to
-implement policies efficiently.
+Flows 1-2, which are installed by default, hold the highest priorities, matching non-new and "tracked" packets and
+forwarding them to table [EgressMetric] to bypass the check from egress rules. This means that if a connection is
+established, its packets go straight to table [EgressMetric], with no other match required. In particular, this ensures
+that reply traffic is never dropped because of an Antrea-native NetworkPolicy or Kubernetes NetworkPolicy rule. However,
+this also means that ongoing connections are not affected if the Antrea-native NetworkPolicy or the Kubernetes
+NetworkPolicy is updated.
+
+Flows 3-6, whose priorities are all 44900, are installed for the egress rule `AllowToDb` in the sample policy. These
+flows are described as follows:
+
+- Flow 3 is utilized to match packets with the source IP address in set {10.10.0.24}, which has all IP addresses of
```

Comment:
It can be removed, but I think it is oaky to reserve it. If I removed all `utilized`, I also need to tweak these lines by moving another word from the next line, and another word to the next line from the next next line, and next....